samples/DotNet/Microsoft.Azure.ServiceBus/BasicSendReceiveUsingQueueClient/Program.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace BasicSendReceiveUsingQueueClient
+{
+    using Microsoft.Azure.ServiceBus;
+    using Microsoft.Azure.ServiceBus.Primitives;
+    using System;
+    using System.Text;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    class Program
+    {
+        const string ServiceBusConnectionString = "{ServiceBus connection string}";
```

Comment:
I do that to my customers and no one has ever complained 😃 
Having a shared config file is also better than multiple locations to edit.